internal/ui/keyboard_js.go

```diff
@@ -0,0 +1,39 @@
+// +build js
+
+package ui
+
+import "github.com/gopherjs/gopherjs/js"
+
+var doc = js.Global.Get("document")
+
+func push(char rune) {
+	go func() {
```

Comment:
Is goroutine needed here? (There are no caller of `push`, so probably this is a temporary hack?)